core/tests/karma-globals.js

```diff
@@ -17,6 +17,32 @@
  */
 
 var GLOBALS = {
+  ALLOWED_INTERACTION_CATEGORIES: '',
+  can_edit: true,
+  can_publish: true,
+  CAN_SEND_EMAILS: true,
+  can_translate: true,
+  canDelete: true,
+  canEdit: true,
+  canModifyRoles: true,
+  canReleaseOwnership: true,
+  canUnpublish: true,
+  collectionId: '',
+  collectionTitle: '',
+  CONTINUOUS_COMPUTATIONS_DATA: '',
+  csrf_token: '',
+  DEFAULT_OBJECT_VALUES: null,
+  DEFAULT_TWITTER_SHARE_MESSAGE_DASHBOARD: '',
+  DEFAULT_TWITTER_SHARE_MESSAGE_EDITOR: '',
+  DEFAULT_TWITTER_SHARE_MESSAGE_PLAYER: '',
+  DEMO_COLLECTIONS: '',
+  DEMO_EXPLORATION_IDS: '',
```

Comment:
Don't some of these need to be empty arrays?

Reply:
This is only to make ts compiler happy - since it searches for all the GLOBALS stuff in karma-globals but that is not used by js. So, I assigned empty values here.